news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,233 @@
+title:  "PrestaShop 1.7.8.0 beta is open for feedback!"
+subtitle: "You can download it and test it"
+date:   2020-05-31 17:00:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - beta
+ - minor
+ - releases
+ - 1.7.8
+---
+
+The first Beta version for PrestaShop 1.7.8.0 is now ready for you to test!
+
+We are very happy to announce today the release of the beta version of 1.7.8.
+This version includes many features and bug fixes, we count on you to test every nook and cranny and report any bugs you encounter.
+
+In the new features: we have the redesign of the translation system that facilitates export and translation of your modules and themes, a new custom header for multistore fans and many other improvements.
+
+//Focus on the contribution (stats)
+
+We need the help of all the community to test this version. It integrates more than 1,372 Pull Request merged and 241 bugs fixed including 17 major bugs and 4 Topwatchers bugs (an issue that are followed by more than 5 people). 
+
+**Remember, this beta version is pre-release software. Do not use it in your production shop!**
+
+## What’s new?
+
+### New features & improvements
+
+#### Translations
+
+- The translation has been revamped, adding the possibility to export Back-Office, Front-Office, eMail and Modules translations.
+
+![the new translation export](/assets/images/2021/translation-export.png)
+
+
+- The Modules can now use the translation system and all their translation can be found on the Translation Page ([link the documentation](link))
+
+
+#### Multistore
+
+- The MultiStore feature embeds a new header. The color of the header can be now customizable for a shop or a group shop in the MultiStore parameters. 
+
+![the new multistore header](/assets/images/2021/multistore_header.png)
+
+It will limit the errors via a colored header and therefore avoid setting up a store thinking it is on another one.
+
+- Only on the maintenance page for now, a drop-down list allows the merchant to view the list of the shop and see if the page has inherited the group setting or has been customized.
+
+![the new multistore specific setting drop down](/assets/images/2021/Specific_settings_drop_down_multistore.png)
+
+It allows merchants to know at a glance which settings are applied in all shops and which ones have been customized to understand the different behaviors of theirs shops.
+
+#### Experimental features
+
+The version 1.7.8 is introducing the Experimental features so that complex new features can be disabled or hidden by default until considered stable. The Experimental features works as a Feature Flags that can allow developers to deliver small incremental chunks without compromising system stability, which is a common practice in the software world. So on Advanced Parameters > Experimental Features, knowing the risks involved, the developers can enable or disable a feature.
+
+![the new Experimental features page](/assets/images/2021/experimental-features.png)
```

Comment:
The screenshot shows a Back Office with some statements in French and some statements in English